photo/tests/test_queries/test_collection.py

```diff
@@ -0,0 +1,115 @@
+from django.test import TestCase
+
+from photo.models import Collection
+from photo.schema import schema
+from photo.tests.factories import CollectionFactory, PictureFactory, UserFactory
+from photo.tests.test_queries.graphql_queries import (
+    collections_query_all,
+    collections_query_filter,
+)
+
+
+class CollectionTest(TestCase):
+    def setUp(self):
+        self.batch = 10
+        user = UserFactory(user_profile_picture=True)
+        self.pictures = PictureFactory.create_batch(self.batch, user=user)
+        self.collections = CollectionFactory.create_batch(
+            self.batch, collection_pictures=self.pictures
+        )
+
+    def test_query_all(self):
+        result = schema.execute_sync(
+            collections_query_all,
+            variable_values={},
+        )
+
+        self.assertEqual(result.errors, None)
+        self.assertEqual(len(result.data["collections"]), self.batch)
+        self.assertEqual(len(result.data["collections"][0]["pictures"]), self.batch)
+        self.assertEqual(
+            sorted([key for key in result.data["collections"][0].keys()]),
+            sorted(
+                [
+                    field.name
+                    for field in (
+                        Collection._meta.fields + Collection._meta.many_to_many
+                    )
+                ]
+            ),
+        )
+
+    def test_query_one(self):
```

Comment:
here we're testing a filter so the name should be test_filter_by_<the filter being tested>